v2/Makefile

```diff
@@ -115,3 +115,7 @@ msgfmt: $(i18n_lua)
 
 $(i18n_lua): $(po_files) $(msgfmt_pl)
 	$(msgfmt_pl) -o $(i18n_lua) $(po_files)
+
+.PHONY: gentemp
+gentemp:
+	./util/gen-templates.js
```

Comment:
You are still using Makefile as a shell. This is not how Makefile works. Makefiles are goal-driven and file-driven. We should only generate those new `.tt2` files when they are not present (or during a force rebuild via `make -B`). The rules should be like:

```make
# add our new tt2 files to the existing tt2_files variable:
tt2_files := $(sort $(wildcard templates/*.tt2)) templates/posts-slide-cn.tt2 templates/posts-slide-en.tt2

templates/posts-slide-cn.tt2 templates/posts-slide-cn.tt2:
﻿﻿﻿﻿﻿﻿﻿﻿    ./util/gen-templates.js
```

Also, the automatically generated `.tt2` files should be cleaned in `make clean`:

```make
.PHONY: clean
clean:
        rm -f templates/posts-slide-*.tt2
        rm -rf html text *.tsv
        rm -f $(templates_lua)
```

Reply:
changed.